russh/src/ssh_read.rs

```diff
@@ -151,13 +152,25 @@ impl<R: AsyncRead + Unpin> SshRead<R> {
                 }
             }
 
+            // We have a full line, check if it is a valid SSH protocol
+            // identifier. The SSH protocol identifier is defined in
+            // https://datatracker.ietf.org/doc/html/rfc4253#section-5.1
+            let ssh_version_regex = match Regex::new(r"^SSH-(1\.99|2\.0)-.*") {
```

Comment:
Random drive by comment that you are free to ignore: Is it really worth pulling in the regex crate for something that could as well be written as `s.starts_with("SSH-1.99-") || s.starts_with("SSH-2.0-")`?

Feel free to answer "yes", but I was wondering...